README.md

```diff
@@ -49,3 +49,7 @@ We expect readers of the documentation to have a basic understanding of Docker a
 
 - [Docker overiew](https://docs.docker.com/get-started/overview/)
 - [Kubernetes overview](https://kubernetes.io/docs/concepts/overview/)
+
+## Discussion
+
+Get engage with us on [Armada slack](https://cloud-native.slack.com/?redir=%2Farchives%2FC03T9CBCEMC)
```

Comment:
```suggestion
If you are interested in discussing Armada you can find us on [Armada slack](https://cloud-native.slack.com/?redir=%2Farchives%2FC03T9CBCEMC).
```

Reply:
Could you also look into how to add github badges for slack?

Reply:
@kannon92 Done and I added the GitHub badge that is redirecting to cncf armada slack discussion's channel as suggested by you.